css/index.css

```diff
@@ -1 +1,264 @@
-/* Compile your LESS file! */
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
Its cool to check out how the less is compiled into css, see what the compiler actually does